proxy/http2/Http2ConnectionState.cc

```diff
@@ -1183,8 +1183,9 @@ Http2ConnectionState::release_stream(Http2Stream *stream)
       // We were shutting down, go ahead and terminate the session
       // this is a member of Http2ConnectionState and will be freed
       // when ua_session is destroyed
-      ua_session->destroy();
-
+      if (ua_session) {
```

Comment:
This looks weird because `ua_session` is already null-checked in above (line #1169).